server/pkg/projects/validator.go

```diff
@@ -2,12 +2,12 @@ package projects
 
 import (
 	"github.com/darwinfroese/hacksite/server/models"
+	validation "github.com/go-ozzo/ozzo-validation"
 )
 
 // ValidateProject checks if the model is valid
-func ValidateProject(project models.Project) bool {
-	// TODO: Find a cleaner validation system
-	valid := project.Name == ""
-
-	return !valid
+func (project models.Project) ValidateProject() error {
```

Comment:
Looks good! Can you have it called in the `CreateProject` and `UpdateProject` functions as well?

Also, same as with accounts this needs to be moved to the `projects` package.

Reply:
lemme change it, 
Hmmm, is it still on the same issue or need to create the new one?

Reply:
The same one is fine since it is applying the validation on the server side.